server/routes/allocationsMigration/allocationsMigrationController.ts

```diff
@@ -63,7 +69,12 @@ export default class AllocationsMigrationController {
       res.redirect('/allocations-migration/amend')
     } else {
       const filter = AllocationsMigrationController.toFilter(req.session.startAllocationsMigrationForm)
-      const count = await this.nomisMigrationService.getAllocationsMigrationEstimatedCount(filter, context(res))
+      const activityCategories = await this.activitiesService.getActivityCategories(context(res))
+      const count = await this.nomisPrisonerService.getAllocationsMigrationEstimatedCount(
+        filter,
+        activityCategories,
+        context(res),
+      )
```

Comment:
This sums up what the PR is all about - get the DPS activity categories from the Activities API and use that to call `hmpps-nomis-prisoner-api` directly rather than calling `hmpps-prisoner-from-nomis-migration` as a proxy to nomis API.